### DISCUSSION TITLE
Is it possible to show the <NavigationMenu/> component Content at all times?

### DISCUSSION THREAD

**Discussion by julicheng** [radix-ui/primitives#1499] [Help] [answered]
I have implemented the `<NavigationMenu/>` component in my app which works great. However, on mobile I would like to style it differently and have the `<NavigationMenu.Content/>` showing at all times when one of the items in the list is the active link. Is this possible? I have tried using `forceMount` which does show the content at all times but the content isn't usable. The aria attributes on the content appear to still be in a state of `closed` and the links have a `tabIndex` of -1.

Is there a possible workaround for this so that I can also use this component on my mobile design?

TIA 🙇🏼‍♀️

**Comment by benoitgrelard** [comment, marked as answer]
Hi @julicheng,

The purpose of `forceMount` is to hand over lifecycle to javascript animation libraries and so our built in animation handling doesn’t support this type of use.

It's on our radar to [provide additional mount options](https://github.com/radix-ui/primitives/discussions/855#discussioncomment-1621945) for cases like this but at the moment it’s not something we support via `forceMount` i'm afraid.

**Reply by julicheng** [reply]
Hi @benoitgrelard,

Thanks for your response. Looking forward to the additional mount options in the future 🙂
